src/cider/nrepl/middleware/util/namespace.clj

```diff
@@ -24,11 +25,15 @@
   (str (System/getProperty "user.dir")
        (System/getProperty "file.separator")))
 
+(defn- re-quote
```

Comment:
I think there's no need for this function. We can just use Pattern/quote directly.